apps/verticalhome/js/pages.js

```diff
@@ -59,6 +59,26 @@
       results.forEach(function(x) {
         var card = new PinCard(x);
         docFragment.appendChild(card.element);
+        card.element.onclick = function launch(data) {
+          if (!data.url) {
+            return;
+          }
+
+          var features = {
```

Comment:
Why is it necessary to set these features? We don't really want to set the name of the window to the page title do we? And where is the icon used?